### PR TITLE
[Feature] Add basic form structure

### PR DESCRIPTION
**Pull request by batatavf2**
> Trello: https://trello.com/c/rP396pnN/1-prot%C3%B3tipo-da-solu%C3%A7%C3%A3o

#### What is the purpose of this pull request?

Add a simple form view, so the user can send a list of emails to be tested, and basic contact info (name, phone number, email).

This PR also:
- Add [bootstrap](https://getbootstrap.com/) static files, to avoid using a lot of custom CSS;
- Include **onboarding** routes in the main urls.py file.
- *Temporarily* disables `django.middleware.csrf.CsrfViewMiddleware`, until one of us figure out how to make this work properly.

#### How should this be manually tested?

Run this branch locally and visit the [onboarding page](http://127.0.0.1:8000/onboarding/).

#### Screenshots or example usage

image

### CODE REVIEW

File: alice/settings.py [hunk 1]
```diff
@@ -37,13 +38,14 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'onboarding'
```
Comment (rodrigombsoares) [inline alice/settings.py:41]: Excelente!

**Review by rodrigombsoares — APPROVED** [review]
LGTM. Nice work! 🚀

File: onboarding/views.py [hunk 2]
```diff
@@ -1,3 +1,8 @@
 from django.shortcuts import render
+from django.template import loader
```
Comment (batatavf2) [inline onboarding/views.py:2]: I think this can be removed 🤔

Comment (rodrigombsoares) [reply onboarding/views.py:2]: Looks like an unused import

File: alice/urls.py [hunk 3]
```diff
@@ -14,8 +14,9 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
     path('admin/', admin.site.urls),
```
Comment (ppedropaulo) [inline alice/urls.py:20]: Poderia deixar como comment por enquanto

**Review by ppedropaulo — APPROVED** [review]

**Review by mgirardin — APPROVED** [review]
sextou, dia de maldade. commita na master e vrau, deploy automático sem passar por homolog, tempo testando é menos tempo fazendo feature que ninguém vai usar, priu priu priu

**Pull request merged**